tests/foreman/cli/test_host.py

```diff
@@ -852,6 +852,33 @@ def test_positive_list(self):
             self.assertGreaterEqual(len(hosts), 1)
             self.assertIn(client.hostname, [host['name'] for host in hosts])
 
+    @tier3
+    def test_positive_list_by_last_checkin(self):
+        """List all content hosts using last checkin criteria
+
+        :id: e7d86b44-28c3-4525-afac-61a20e62daf8
+
+        :expectedresults: Hosts are listed for the given time period
+
+        :BZ: 1285992
+
+        :CaseLevel: System
+        """
+        with VirtualMachine(distro=DISTRO_RHEL7) as client:
+            client.install_katello_ca()
+            client.register_contenthost(
+                self.new_org['label'],
+                lce='{}/{}'.format(
+                    self.new_lce['label'], self.promoted_cv['label']),
+            )
+            self.assertTrue(client.subscribed)
+            hosts = Host.list({
+                'search': 'last_checkin = "Today" or '
```

Comment:
perhaps we should add the organization to limit the result only to this TestCase organization (as understand this should return all the hosts created on sat from all the tests), what do you think?

Reply:
I think current approach is fine. Adding organization will not speed up our run or prove something more. I would like to test exactly what is needed and do not add extra search criteria